File: ModManager4/Class/InstalledMod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ModManager4.Class
{
    public class InstalledMod
    {
        public string id { get; set; }
        public string version { get; set; }
        public string gameVersion { get; set; }
        public List<string> files { get; set; }

        public InstalledMod(string id, string version, string gameVersion, List<string> files)
        {
            this.id = id;
            this.version = version;
            this.gameVersion = gameVersion;
            this.files = files;
        }
    }
}

File: electron/main/class/installedMod.ts
export class InstalledMod {
    modId: string;
    version: string | null;
    releaseVersion: string | null;

    constructor(modId: string, version: string | null, releaseVersion: string | null) {
        this.modId = modId;
        this.version = version;
        this.releaseVersion = releaseVersion;
    }
}